docs/using-the-compiler.rst

```diff
@@ -301,3 +301,10 @@ Error types
 11. ``CompilerError``: Invalid use of the compiler stack - this should be reported as an issue.
 12. ``FatalError``: Fatal error not processed correctly - this should be reported as an issue.
 13. ``Warning``: A warning, which didn't stop the compilation, but should be addressed if possible.
+
+Decompiler
```

Comment:
Hm, I think we need to find a better chapter for this. Using the compiler section is quite specific in how `solc` works as opposed to "language Solidity". The compiler has more to do with the language, than `solc`.

Reply:
Hey @axic after discussions yesterday with @chriseth we didn't seem to think that a new page or section on decompiling was necessary, but I updated the text based on that discussion.

As for a new location for the section, where do you think? Maybe _contracts.rst_?

Reply:
I didn't mean section on decompiling, but rather moving it to a place for language vs. detail on solc. Why not just put into the "Solidity Tools" subsection? There's a "evmdis" link there.

Reply:
@axic OK will do, in this case, I'll add it to a new PR, and I guess this one is done.